.github/actions/docker-build-and-push/action.yaml

```
@@ -99,23 +109,29 @@ runs:
         push: ${{ inputs.allow-push == 'true' }}
         files: |
           docker/docker-bake.hcl
+          ${{ steps.meta-base.outputs.bake-file }}
           ${{ steps.meta-prebuilt.outputs.bake-file }}
           ${{ steps.meta-devel.outputs.bake-file }}
           ${{ steps.meta-runtime.outputs.bake-file }}
         provenance: false
+        targets: |
+          ${{ inputs.bake-images }}
         set: |
           ${{ inputs.build-args }}
 
-    - name: Build only
+    - name: Build Only
```

Please focus on the minimum change. If you want to refactor please make another PR.
```suggestion
    - name: Build only
```